modules/platform/nuxeo-platform-document-routing/nuxeo-routing-rest-api/src/test/java/org/nuxeo/ecm/restapi/server/WorkflowEndpointTest.java

```diff
@@ -1239,11 +1239,16 @@ public void testTaskWorkflowInfo() throws IOException {
             JsonNode initiatorNode = taskNode.get("workflowInitiator");
             assertEquals("user", initiatorNode.get("entity-type").textValue());
             assertEquals("Administrator", initiatorNode.get("id").textValue());
-            assertTrue(initiatorNode.get("isAdministrator").booleanValue());
             JsonNode properties = initiatorNode.get("properties");
             ArrayNode groups = (ArrayNode) properties.get("groups");
-            assertEquals(1, groups.size());
-            assertEquals("administrators", groups.get(0).textValue());
+            if(!initiatorNode.get("isPartial").booleanValue()) {
```

Comment:
Space before `(`. Also after the other changes below `initiatorNode.get("isPartial")` may be null so this must be checked.